server/lib/src/plugins/refint.rs

```diff
@@ -242,9 +364,10 @@ impl ReferentialIntegrity {
                 c.attribute_equality(Attribute::Class.as_ref(), &EntryClass::DynGroup.into());
 
             ref_types.values().filter_map(move |rtype| {
-                // Skip dynamic members
+                // Skip dynamic members, these are recalculated by the
+                // memberof plugin.
                 let skip_mb = dyn_group && rtype.name == "dynmember";
-                // Skip memberOf.
+                // Skip memberOf, also recalculated.
                 let skip_mo = rtype.name == "memberof";
```

Comment:
Since we're messing around in this part of the code, can you please use the static objects I added the other day instead of strings for memberof/dynmember?

Reply:
Opps mybad I'll fix that up next time :)